slasher/rpc/server.go

```diff
@@ -51,3 +69,73 @@ func (ss *Server) IsSlashableAttestation(ctx context.Context, req *ethpb.Indexed
 func (ss *Server) IsSlashableBlock(ctx context.Context, req *ethpb.SignedBeaconBlockHeader) (*slashpb.ProposerSlashingResponse, error) {
 	return nil, errors.New("unimplemented")
 }
+
+func (ss *Server) verifyIndexedAttestationSig(ctx context.Context, indexedAtt *ethpb.IndexedAttestation) (bool, error) {
```

Comment:
not sure where should this function live

Reply:
Is this duplicated logic from the beacon node? We should likely find some shared package to place it if that's the case

Reply:
moved it to attestationsutils